frontend/src/components/CreatorCard.jsx

```diff
@@ -55,31 +55,11 @@ const CreatorCard = ({ creator }) => {
 
     // <Link to={`/creator/${creatorId}`}>
     // add ig profile pic
-<<<<<<< HEAD
-    <Link to={`/creators/${creator.id}`}>
-      <Container>
-        <h4>{creator.fullName}</h4>
-        <Image src="#" alt="profile photo" />
-        <TextBox>{creator.description} [This will be Description.]</TextBox>
-        <EngagementList>
-          <Engagement id="followers">
-            <FollowerIcon />
-            [ig followers]
-          </Engagement>
-          <Engagement id="impressions">
-            <ImpressionIcon />
-            [ig impressions]
-          </Engagement>
-          <Engagement id="location">
-            <LocationIcon /> [ig location]
-          </Engagement>
-        </EngagementList>
-        <TextBox id="interests"> [interest list] </TextBox>
-      </Container>
-    </Link>
-=======
+
     <Container>
-      <h4>{creator.fullName}</h4>
+      <Link to={`/creators/${creator.id}`}>
+        <h4>{creator.fullName}</h4>
+      </Link>
```

Comment:
How about we have this individual creator link for each creatorCard?
`<Container>
      <Link to={`/creators/${creator.id}`}>
        <h4>{creator.fullName}</h4> ...
      </Link>
    </Container>`